jdk/src/share/back/classTrack.c

```diff
@@ -81,13 +87,13 @@ cbTrackingObjectFree(jvmtiEnv* jvmti_env, jlong tag)
 struct bag *
 classTrack_processUnloads(JNIEnv *env)
 {
+    struct bag* deleted = deletedSignatures;
```

Comment:
Reading `deletedSignatures` must be within the locked region.

Reply:
Fixed.